### PR TITLE
feat: Update package for cloud run

### PR DESCRIPTION
**Pull request by gbouv**

### CODE REVIEW

File: validator.star
Comment (Peeeekay) [inline validator.star:28]: Oh we are using custom image until they hyperlane stuff gets merged to monorepo ?

**Review by Peeeekay — APPROVED** [review]
Looks good.

File: validator.star
Comment (gbouv) [reply validator.star:28]: Yes we don't have the choice, waiting for this one to get merged but it's taking ages
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2647

In the meantime I've been building it locally and pushing to my dockerhub space

I'll make sure to update this once my PR gets merged and the new images are published

**Pull request merged**
